src/pages/events/index.svelte

```diff
@@ -1,90 +1,164 @@
+
+      // let stateObj = { id: "100" };
+      // window.history.replaceState(stateObj, "Events", "/events");
+
+    }
+
+    return "";
+  }
 </script>
 
 <style>
-.event-buttons-container .btn {
-  width: 10em;
-}
-
-.event-buttons-container .btn:disabled {
-  cursor: unset;
-}
-
-.logo {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-}
-
-/* these will get used only on XS screens */
-.logo img {
-  max-height: 5rem;
-  margin-bottom: 2rem;
-}
-li {
-  max-height: initial;
-}
-
-/* https://getbootstrap.com/docs/4.4/layout/overview/#responsive-breakpoints */
-/* these will get used on everything above XS screens using bootstrap's breakpoint settings */
-@media (min-width: 576px) {
+  @keyframes event-blinking {
+    25% {
+      border-color: #dee2e6;
+    }
+    43.75%,
+    81.25% {
+      border-color: #85888a;
+    }
+    62.5%,
+    100% {
+      border-color: #dee2e6;
+    }
+  }
+
+  .blinking {
+    animation: event-blinking 4s;
+  }
+
+  .event-item {
+    border-style: solid;
+    border-width: 1px;
+    border-color: #dee2e6;
+  }
+
+  .event-buttons-container .btn {
+    width: 10em;
+  }
+
+  .event-buttons-container .btn:disabled {
+    cursor: unset;
+  }
+
+  .logo {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  /* these will get used only on XS screens */
   .logo img {
-    max-width: 90%;
-    max-height: 9rem;
-    margin-bottom: initial;
+    max-height: 5rem;
+    margin-bottom: 2rem;
   }
   li {
-    max-height: 11rem;
+    max-height: initial;
+  }
+
+  /* https://getbootstrap.com/docs/4.4/layout/overview/#responsive-breakpoints */
+  /* these will get used on everything above XS screens using bootstrap's breakpoint settings */
+  @media (min-width: 576px) {
+    .logo img {
+      max-width: 90%;
+      max-height: 9rem;
+      margin-bottom: initial;
+    }
+    li {
+      max-height: 11rem;
+    }
   }
-}
 </style>
 
 <FilteredDisplay title="Events" filter={eventFilter} items={$events}>
   <div slot="filters">
     <EventFilters filter={eventFilter} />
   </div>
   <div slot="items" let:items={filteredEvents}>
-    <button on:click={() => $goto("/events/new")} type="submit" class="btn btn-primary mb-2">Create Event</button>
+    <button
+      on:click={() => $goto("/events/new")}
+      type="submit"
+      class="btn btn-primary mb-2">Create Event</button
+    >
     {#if $loading}
       <p>⏳ Retrieving events...</p>
-    {:else if filteredEvents.length }
+    {:else if filteredEvents.length}
+      {scrollToEvent(filteredEvents)}
       <ol class="list-unstyled">
-        {#each filteredEvents as event (event.id) }
-          <li class="border rounded mb-2 p-2"
-              animate:flip="{{ duration: 350 }}"
-              in:fade>
+        {#each filteredEvents as event, i (event.id)}
+          <li
+            id="event-{i + 1}"
+            class="rounded mb-2 p-2 event-item {$params.scrollTo &&
```

Comment:
did `border` get lost along the way?

Reply:
I removed border because it was forcing the border color, and I needed to have the ability to change it (highlighted new event). The `event-item` class has the same function, just without the `!important`.